ivy/data_classes/array/experimental/losses.py

```diff
@@ -1,6 +1,102 @@
 # global
 import abc
+import ivy
+from typing import Optional, Union
 
 
 class _ArrayWithLossesExperimental(abc.ABC):
-    pass
+    def mse_loss(
+        self: ivy.Array,
+        pred: Union[ivy.Array, ivy.NativeArray],
+        /,
+        *,
+        axis: Optional[int] = None,
```

Comment:
This will fail the test as axis argument isn't defined in the original function

Reply:
Thank you very much @hirwa-nshuti for your valuable feedback, I look forward to correcting this as per your feedback. Thanks for your patience in this PR, throughout :smile:

Reply:
Updated the original functions with axis argument, @hirwa-nshuti. Let's see.